lib/ansible/module_utils/common/arg_spec.py

```diff
@@ -114,6 +106,31 @@ def __init__(self, argument_spec,
                  required_by=None,
                  ):
 
+        """
+        :param argument_spec: Specification of valid parameters and their type. May
+            include nested argument specs.
+        :type argument_spec: dict
+
+        :param mutually_exclusive: List or list of lists of terms that should not
+            be provided together.
+        :type mutually_exclusive: list
+
+        :kwarg required_together: List of lists of terms that are required together.
+        :type required_together: list
```

Comment:
Or maybe
```suggestion
        :type required_together: list or None
```
per https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#python-signatures